common/app/views/mainAMP.scala.html

```diff
@@ -3,6 +3,7 @@
 @import common.{AnalyticsHost, CanonicalLink, LinkTo}
 @import conf.Configuration
 @import views.support.OmnitureAnalyticsData
+@import views.support.FBPixel
```

Comment:
Does it make more sense to write this as:
`@import views.support.{OmnitureAnalyticsData, FBPixel}`